CONTRIBUTING.md

```diff
@@ -1,54 +1,82 @@
+
+There are many ways to contribute: writing tutorials or blog posts about your experience, improving the [documentation](https://github.com/artefactual/archivematica-docs/), submitting bug reports, answering questions on the [mailing list](https://groups.google.com/forum/#!forum/archivematica), or writing code which can be incorporated into Archivematica itself.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
-- [Code review](#code-review)
+- [Submitting bugs](#submitting-bugs)
+- [Submitting code changes](#submitting-code-changes)
   - [Getting started](#getting-started)
   - [When to submit code for review?](#when-to-submit-code-for-review)
   - [Opening the pull request](#opening-the-pull-request)
   - [Discussion](#discussion)
   - [Cleaning up the commit history](#cleaning-up-the-commit-history)
-    - [Commits should be specific and atomic](#commits-should-be-specific-and-atomic)
-    - [Every commit should work](#every-commit-should-work)
-    - [Commit summaries should be concise](#commit-summaries-should-be-concise)
-    - [Commit messages should be as detailed as they need to be (and no more)](#commit-messages-should-be-as-detailed-as-they-need-to-be-and-no-more)
+- [Contributor's Agreement](#contributors-agreement)
+  - [Why do I have to sign a Contributor's Agreement?](#why-do-i-have-to-sign-a-contributors-agreement)
+  - [How do I send in an agreement?](#how-do-i-send-in-an-agreement)
+  - [Small changes](#small-changes)
 - [Contribution standards](#contribution-standards)
   - [Style](#style)
     - [Some extra notes](#some-extra-notes)
     - [Exceptions](#exceptions)
     - [Working with legacy code](#working-with-legacy-code)
   - [Documentation](#documentation)
   - [Tests](#tests)
-- [Contributor's Agreement](#contributors-agreement)
-  - [Why do I have to sign a Contributor's Agreement?](#why-do-i-have-to-sign-a-contributors-agreement)
-  - [How do I send in an agreement?](#how-do-i-send-in-an-agreement)
+  - [Commit History](#commit-history)
+    - [Commits should be specific and atomic](#commits-should-be-specific-and-atomic)
+    - [Every commit should work](#every-commit-should-work)
+    - [Commit summaries should be concise](#commit-summaries-should-be-concise)
+    - [Commit messages should be as detailed as they need to be (and no more)](#commit-messages-should-be-as-detailed-as-they-need-to-be-and-no-more)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Code review
+## Submitting bugs
+
+If you find a security vulnerability, do NOT open an issue.
+Email info@artefactual.com instead.
+
+We accept bugs submitted through Github Issues or through our [technical](https://groups.google.com/forum/#!forum/archivematica-tech) or [user](https://groups.google.com/forum/#!forum/archivematica) mailing lists.
```

Comment:
How about something like this, for line 44:

Artefactual staff use the issue tracker at https://projects.artefactual.com/projects/archivematica, for any work they do on the Archivematica project.  Anyone is welcome to request an account on that system and file issues there. 

Issues can also be filed using the GitHub Issues page of the Archivematica project, or any of the supporting github repositories in the Artefactual organization.  You can also post in one of our discussion groups, at 
 [technical](https://groups.google.com/forum/#!forum/archivematica-tech) or [user](https://groups.google.com/forum/#!forum/archivematica).